sources/api/schnauzer/src/helpers.rs

```diff
@@ -2362,3 +2397,74 @@ mod test_etc_hosts_helpers {
         assert_eq!(result, "")
     }
 }
+
+#[cfg(test)]
+mod test_any_enabled {
+    use super::*;
+    use handlebars::RenderError;
+    use serde::Serialize;
+    use serde_json::json;
+    use std::collections::BTreeMap;
+
+    // A thin wrapper around the handlebars render_template method that includes
+    // setup and registration of helpers
+    fn setup_and_render_template<T>(tmpl: &str, data: T) -> Result<String, RenderError>
+    where
+        T: Serialize,
+    {
+        let mut registry = Handlebars::new();
+        registry.register_helper("any-enabled", Box::new(any_enabled));
+
+        let mut input_data = BTreeMap::new();
+        input_data.insert("input", data);
+
+        registry.render_template(tmpl, &input_data)
+    }
+
+    const TEMPLATE: &str = r#"{{#if (any-enabled input)}}enabled{{else}}disabled{{/if}}"#;
+
+    #[test]
+    fn test_any_enabled_with_enabled_elements() {
+        let result = setup_and_render_template(TEMPLATE, &json!([{"enabled": true}])).unwrap();
```

Comment:
Seems like it would be good to prove that we don't care what else is in the object.

```suggestion
        let result = setup_and_render_template(TEMPLATE, &json!([{"foo": [], "enabled": true}])).unwrap();
```